src/main/java/md/miller1995/Dealership/models/dto/CarDTO.java

```diff
@@ -2,9 +2,11 @@
 
 import java.io.Serializable;
 import java.time.LocalDate;
+import java.util.List;
 
 
 public class CarDTO implements Serializable {
+
```

Comment:
Maybe the use of Lombok to get rid of boilerplate code for getters & setters, constructors and so on?

Reply:
ok, I changed code in this class using Lombok.